vignettes/coef_vcov.Rmd

```diff
@@ -52,7 +52,7 @@ Where $W$ is the block diagonal matrix of inverse of covariance matrix of $\epsi
 
 ## Empirical Covariance
```

Comment:
Can we also in this vignette include the syntax for how to request this in `mmrm`?